sdk/entra/WebJobs.Extensions.AuthenticationEvents/test/WebJobs.Extensions.AuthenticationEvents.Tests.csproj

```diff
@@ -0,0 +1,72 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
+		<IsTestProject>true</IsTestProject>
+		<SignAssembly>True</SignAssembly>
```

Comment:
We should remove this; the repository infrastructure will handle this.

Reply:
Will the public key remain the same, as internal of the main library are exposed to the test project?

Reply:
It will change to using the standard repository key.   You'll want to update your `InternalsVisibleTo` to the following:

`PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4`

- [Example](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/src/Properties/AssemblyInfo.cs#L7)